shared/android/app/src/main/java/io/keybase/ossifrage/MainActivity.java

```diff
@@ -230,15 +231,19 @@ public void emit() {
       }
 
       // Closure like class so we can keep our emit logic together
-      class Emit implements Runnable {
+      class Emit {
```

Comment:
Now that the rest of the logic is in `IntentEmitter`, do we still need this closure? [you don't have to address in this PR]